src/env/args.js

```diff
@@ -2,7 +2,13 @@ require('dotenv').config();
 
 const minimist = require('minimist');
 
-const args = minimist(process.argv.slice(2), {
+const runner = process.argv[1].split('/').slice(-1)[0]
+const args = runner === 'kopytko' ? process.argv.slice(3) : process.argv.slice(2);
```

Comment:
Can you add a comment on how argv looks like for `kopytko` as a runner, and how it looks like if a runner is not `kopytko`?